derive/src/utils.rs

```diff
@@ -99,7 +99,7 @@ where
     find_meta_item("codec", itr, pred)
 }
 
-fn find_meta_item<'a, F, R, I, M>(kind: &str, mut itr: I, mut pred: F) -> Option<R>
+pub(crate) fn find_meta_item<'a, F, R, I, M>(kind: &str, mut itr: I, mut pred: F) -> Option<R>
```

Comment:
Nit: rather than making this `pub(crate)`, perhaps expose another util like the above which looks for `#[scale_info(rename = "...")]` and returns something like `Option<&str>` if a rename happened or `None` if not?